messaging/methods/messages/getMessages.md

```diff
@@ -31,6 +31,17 @@ Authentication on this endpoint is <b>NOT</b> done via API token and secret. Ins
 | after | integer | The record number indicates that search result will have records after the given position number in a order of sorted received timestamp | `after=10 indicates that search result will have the records after 10th postion in a order of sorted received timestamp` |
 | limit | integer | The maximum records requested in search result . Default `100`. <br> The sum of limit and after cannot be more than 10000 | `limit=100` |
 
+### Message Statuses
+
+| Status | Description |
+|:--|:--|
+| RECEIVED | Message has been received by bandwidth. |
+| QUEUED | Message has been queued prior to sending. |
+| SENDING | Message is in the process of being sent. This is a temporary status. |
+| SENT | Message has been sent. |
+| DLR_TIMEOUT | |
```

Comment:
Message was forwarded and then did not receive any DLR within a certain amount of time that it would have expected to get one. I think the timeout is 2 hours, idk if this doc should specify a specific time, which could change, or just say that there was some timeout

Reply:
Works for me. Thank you!

Reply:
For now I'll just keep your first sentence, if people ask for specific time we can put it (maybe more appropriate for GA release)